### ISSUE TITLE
Crash on Sonic Visualiser 5.0

### ISSUE THREAD

**Issue by w4iei** [cannam/expressive-means#32]
Hi, I'm excited to try expressive-means with Sonic Visualiser 5.0, but it crashes immediately upon trying to load any expressive-means transform.

**Comment by w4iei** [comment]
Additional details. I'm on ARM MacOS 14.6.1.

https://github.com/user-attachments/assets/33810ae8-fb39-4159-8b67-781540a79adf

**Comment by w4iei** [comment]
I have just confirmed that the plugin works without crashing on my machine with Sonic Visualiser 4.5.2, without modifying any other files besides the .app

**Comment by cannam** [comment]
Thanks for the report!

Can I just check: Did you install Expressive Means using the package attached to the [release page of this repository](https://github.com/cannam/expressive-means/releases/tag/v1.0.0), or did you install it using the [Vamp Plugin Pack](https://vamp-plugins.org/pack.html)?

(The Vamp Plugin Pack was recently updated to include Expressive Means - previously it was only available from this repo. The plugin should work the same either way, I'd just like to gather as much info as possible.)

**Comment by w4iei** [comment]
Yes, I installed it by copy/pasting the three files from the .dmg. I did not use the Vamp Plugin Pack.

**Comment by cannam** [comment]
Can you try the new 5.0.1 release of Sonic Visualiser? I am taking a wild guess that the crash is actually the same one as the rather arbitrary Qt bug reported in this Sonic Visualiser issue https://github.com/sonic-visualiser/sonic-visualiser/issues/94.
